democracy/migrations/0051_auto_20200225_1349.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.10 on 2020-02-25 13:49
+
+import django.core.files.storage
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('democracy', '0050_add_on_deletes'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='sectionfile',
+            name='file',
+            field=models.FileField(max_length=2048, storage=django.core.files.storage.FileSystemStorage(location='/home/santtua/kerrokantasi/protected_media'), upload_to='files/%Y/%m', verbose_name='file'),
```

Comment:
This path doesn't look correct? It contains at least the developer's username.

Reply:
Interesting finding. This path doesn't seem to have any real world effect. Whoever makes migrations last will have the storage path set to their local path or env setting.
During runtime storage path is set according to system's env setting.

Both live and test instances of kerrokantasi have always been running this storege path setting with someone's dev path.

Reply:
Agreed that these are already earlier merged so at this point we don't do fixes.